app/javascript/mastodon/locales/locale-data/Mastodon-cs-data.txt

```diff
@@ -0,0 +1,138 @@
+/*eslint eqeqeq: "off"*/
```

Comment:
[`locale-data/cs.js`](https://unpkg.com/react-intl@2.4.0/locale-data/cs.js) https://unpkg.com/react-intl@2.4.0/locale-data/cs.js

Reply:
Also the file is named `Mastodon-cs-data.txt`

Reply:
Yeah, was uploading that and forgot to rename it.